packages/kit/scripts/extract-types.js

```diff
@@ -39,7 +39,9 @@ for (const statement of node.statements) {
 			trailingComma: 'none'
 		});
 
-		types.push({ name, comment, snippet });
+		if (snippet) {
```

Comment:
what's a situation in which `snippet` would be falsy (other than the bug described above)?

Reply:
reverted this as well and updated the script to handle non-exported types